core/commonMain/src/kotlinx/serialization/encoding/AbstractEncoder.kt

```diff
@@ -40,6 +40,10 @@ public abstract class AbstractEncoder : Encoder, CompositeEncoder {
         throw SerializationException("'null' is not supported by default")
     }
 
+    public open fun skipNullElement(descriptor: SerialDescriptor, index: Int): Boolean {
```

Comment:
This looks like a json-specific method, so it shouldn't be in `AbstractDecoder` (moreover, third-party formats use this skeleton class too)

Reply:
I think it's better to override `encodeNullableSerializableElement` directly in JSON

Reply:
`encodeNullableSerializableElement` is a `final` in `AbstractDecoder` that's why I did add `skipNullElement` function.
Make `AbstractDecoder.encodeNullableSerializableElement` `open`?

Reply:
I think we can decide this on the next design meeting